neo/Persistence/LevelDB/DbMetaDataCache.cs

```diff
@@ -37,5 +38,14 @@ protected override void UpdateInternal(T item)
         {
             batch?.Put(new byte[] { prefix }, item.ToArray());
         }
+
+        public override void Dispose()
+        {
+            if (options != null)
+            {
+                MethodInfo method = options.GetType().GetMethod("FreeUnManagedObjects", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
it haven't got Dispose? this reflection is very slow

Reply:
Oh yes, I made a mistake.